Library/Homebrew/os/mac.rb

```diff
@@ -41,8 +41,12 @@ def cat
       version.to_sym
     end
 
+    def languages
+      @languages ||= Utils.popen_read("defaults", "read", ".GlobalPreferences", "AppleLanguages").scan(/[^ \n"(),]+/)
+    end
+
     def language
-      @language ||= Utils.popen_read("defaults", "read", ".GlobalPreferences", "AppleLanguages").delete(" \n\"()").sub(/,.*/, "")
+      languages.first
```

Comment:
> there's a slim (or even zero?) chance of this macOS setting to be set to an empty array

There is only one way I can think of this could fail: Manually changing/deleting `.GlobalPreferences`, which is highly unlikely, so I think we're good here.